### PR TITLE
minor fixes to the Istio Standard Metrics page

### PR DESCRIPTION
**Pull request by NisheekaNynan1**
Please provide a description for what this PR is for.

- Added a fix as https://github.com/istio/proxy/tree/master/extensions/stats is a broken link - 404
- Added backticks instead of double quotes for `unknown` and `details` under the `Labels` section

And to help us figure out who should review this PR, please 
put an X in all the areas that this PR affects.

- [ ] Configuration Infrastructure
- [x] Docs
- [ ] Installation
- [ ] Networking
- [ ] Performance and Scalability
- [ ] Policies and Telemetry
- [ ] Security
- [ ] Test and Release
- [ ] User Experience
- [ ] Developer Infrastructure

### CODE REVIEW

File: content/en/docs/reference/config/metrics/index.md [hunk 1]
```diff
@@ -11,7 +11,7 @@ aliases:
 
 The following are the standard service level metrics exported by Istio.
 
-The telemetry component is [implemented](https://github.com/istio/proxy/tree/master/extensions/stats) as a [Proxy-wasm](https://github.com/proxy-wasm/spec) plugin.
+The telemetry component is implemented as a [Proxy-wasm](https://github.com/proxy-wasm/spec) plugin.
```
Comment (zirain) [inline content/en/docs/reference/config/metrics/index.md:14]: this's not correct now, stats is a [native(c++) filter](https://github.com/istio/proxy/tree/master/source/extensions/filters/http/istio_stats).
cc @kyessenov

Comment (kyessenov) [reply content/en/docs/reference/config/metrics/index.md:14]: Yeah, I'd say that components are implemented as "proxy extensions" without going into detail. MX is still Wasm, but it's really not important to a user of Istio, as long as it works.

Comment (NisheekaNynan1) [reply content/en/docs/reference/config/metrics/index.md:14]: Do you suggest I change line 14 to `The telemetry component is implemented as a proxy extension` ? @kyessenov

Comment (NisheekaNynan1) [reply content/en/docs/reference/config/metrics/index.md:14]: Hello, please let me know if there are any changes required, if any. If not I will probably mark this PR as closed if it's not needed.

Comment (zirain) [reply content/en/docs/reference/config/metrics/index.md:14]: yes, that would be better.

Comment (zirain) [inline content/en/docs/reference/config/metrics/index.md:14]:
```suggestion
The telemetry component is implemented as a [Proxy extension](https://github.com/istio/proxy/tree/master/source/extensions/filters/http/istio_stats).
```

Comment (NisheekaNynan1) [reply content/en/docs/reference/config/metrics/index.md]: Hmm I'm trying to figure out what's going on with my EasyCLA auth, please bear with me.

Comment (NisheekaNynan1) [reply content/en/docs/reference/config/metrics/index.md]: All good with the EasyCLA :) sorry about that!

**Review by zirain — APPROVED** [review]
LGTM, thanks!

cc @istio/wg-docs-maintainers-english

**Review by kfaseela — APPROVED** [review]

**Pull request merged**